orchestra/utils/task_lifecycle.py

```diff
@@ -1006,3 +1013,76 @@ def create_subsequent_tasks(project):
                     settings.MACHINE_STEP_SCHEDULER[1])
                 machine_step_scheduler = machine_step_scheduler_class()
                 machine_step_scheduler.schedule(project.id, step.slug)
+
+
+def request_workers_for_task(task, required_role):
```

Comment:
shouldn't this logic live inside of the bot? so we can subclass and swap out how we ping people?